adoptingerlang.org

```diff
@@ -3997,14 +3997,85 @@ No business works only with code; it is way more frequent to hear about projects
+
+Regardless of how you run things at that point, make sure that from time to time your expert sits down with the rest of the team, and asks them about the various points of friction they have. Figure out what annoys them about the toolings, the language, and so on. This will help everyone re-calibrate and improve things. It's otherwise frequent for newcomers to just assume "this is how it is" and they may end up living with far more annoyances than they otherwise would.
+
 *** Building Without an Expert
+
+Building without an expert is definitely a bit trickier. It's not impossible (some people had to become experts on their own to get going) and there are a lot of resources out there—books, talks, forums and chat rooms, and so on—but it's definitely more of a challenge and it can be costlier. The biggest risk in building without an expert is that you may not know how long you're going to spend in wrong paths and bad avenues, and the bad learning you do early can stick for a longer time.
+
+This is, ultimately, not too different from regular development, but you'll be headed that way willingly. As such, you may want to structure things a bit differently. Rather than picking one expert and your enthusiastic folks, you will want to explode the requirements and roles across everyone, to try and get the dynamics than an expert would have facilitated.
+
+Identify people who have prior knowledge of functional languages; they won't know everything, but they already have a head start. Also identify some self-directed learners, those who enjoy sitting down and digging into something. Their job will be to read books and figure things out a few weeks ahead of everyone else; they're quire literally the piano teacher only ahead of you by a couple of weeks. You'll want to find someone who enjoys teaching, writing documentation, or preparing demos. That's often pretty hard, so if you can't find anyone, assign a rotating schedule to that responsibility. Put them in charge of doing broader information dissemination, maintaining docs, and a quick "getting started" track to what you do in your organisation.
+
+Have everyone do bits of development and getting used to the feel of the language, getting involved with things in the community, and leverage the code already existing out there. Here are a few ideas:
+
+- look into existing code out there, and do a show-and-tell of various libraries and their approaches; dig to figure out how they are structured.
```

Comment:
```suggestion
- look into existing code, and do a show-and-tell of various libraries and their approaches; dig to figure out how they are structured.
```

Dropping the "out there" just because it was in the last sentence as well.